frontend/packages/git-service/src/utils/devfile-parser.ts

```diff
@@ -0,0 +1,25 @@
+import { CommandEntry, parse } from 'docker-file-parser';
+
+export class DevfileParser {
+  devfileContent: string;
+
+
+  parsedCommands: CommandEntry[];
+
+  constructor(devfileContent: string) {
+    this.devfileContent = devfileContent;
+    // this.parsedCommands = this.parse();
+  }
+
+  parse(): CommandEntry[] {
+    return parse(this.devfileContent);
+  }
+
+    getDevfileVersion(devfileContent: string): string {
+        this.devfileContent = devfileContent;
+        let version = this.devfileContent.split("\n",1)[0].split(" ")[1];
+        return version;
+    }
```

Comment:
@andrewballantyne just a heads up, there are probably better ways to extract the version, we had just put this here as a placeholder at the time

Reply:
IMO Console shouldnt need to do validation for devfile versions. If the imported devfile is a version that does not support Build Guidance and Console requests for it; then the library should say so and err out. 🤔

Reply:
Yeah, I still need to chase down the various issues with this PR (lint for one) and apply the needed updates. Hoping to do it before our meeting this afternoon. I agree with the not having the version be locked on the client and using the lib as that guidance.

Reply:
@maysunfaisal I think that was discussed in the original plans for it, but since we didn't have the library functioning we just added the check for it in console. It can be removed if there's checks and responses for it in the parser library

Reply:
Dropped this, will send the data to the backend and let that response come back.